crates/sui-types/src/message_envelope.rs

```diff
@@ -191,8 +207,49 @@ where
         self.auth_signature.epoch
     }
 
-    pub fn verify_signature(&self, committee: &Committee) -> SuiResult {
-        self.data.verify(Some(self.auth_sig().epoch))?;
+    pub fn verify_committee_sigs_only(&self, committee: &Committee) -> SuiResult
+    where
+        <T as Message>::DigestType: PartialEq,
+    {
+        self.data.verify_epoch(self.auth_sig().epoch)?;
+        self.auth_signature
+            .verify_secure(self.data(), Intent::sui_app(T::SCOPE), committee)
+    }
+}
+
+impl<T> Envelope<T, AuthoritySignInfo>
+where
+    T: Message + AuthenticatedMessage + Serialize,
+{
+    pub fn verify_signatures_authenticated(
+        &self,
+        committee: &Committee,
+        verify_params: &VerifyParams,
+    ) -> SuiResult {
+        self.data.verify_epoch(self.auth_sig().epoch)?;
+        self.data.verify_message_signature(verify_params)?;
```

Comment:
The difference is that verify_authenticated verifies user signatures, which only applies to Transactions - no other message type currently has an internal verification step like this.

The intent is to allow us to provide VerifyParams only when it is necessary, rather than having to provide `VerifyParams::default()` or whatever when verifying other message types such as CheckpointSummary.

If you have a suggestion for better method names I would be interested!